sdk/cosmos/azure-cosmos-spark_3-5_2-12/CONTRIBUTING.md

```diff
@@ -0,0 +1,84 @@
+# Contributing
+This instruction is guideline for building and code contribution.
+
+## Prequisites
+- JDK 11 and above
```

Comment:
I see we have a test for JDK 8, I am curious, why do we need 11 and above?

Reply:
I just tryed it with jdk 8, so it should be fine to say jdk 8 and above. I will update the other spark modules too.